assets/src/components/customers/CustomerDetailsNewNoteInput.tsx

```diff
@@ -41,6 +41,12 @@ const CustomerDetailNewNoteInput = ({
     setIsSaving(false);
   };
 
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLTextAreaElement>) => {
+    if (e.keyCode === 13 && e.metaKey) {
```

Comment:
nit -- slight preference for `e.key === 'Enter'`, since i think `keyCode` is being deprecated 🤔 https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
🤯

I've been out of web for too long. I didn't even know.